qa/suites/fs/nfs/cluster/1-node.yaml

```diff
@@ -1,11 +1,16 @@
 meta:
 - desc: 1 ceph cluster with 1 mon, 1 mgr, 3 osds, 2 mds, 1 client
```

Comment:
nit: 1 mds?

Reply:
the default placement count for `ceph orch apply mds a` will be 2